# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/events/test/pointerevents/mochitest.toml
# Commit: f8a2a169b296
# Full Hash: f8a2a169b2969d29e04733ae27a56459e5359973
# Author: Alexandru Marc <amarc@mozila.com>
# Date: 2025-05-16 09:19:19
# Regressor Bug: 1550462
# File Overlap Count: 1
# Description:
#   Revert "Bug 1550462 - part 3: Enable `pointerrawupdate` event r=smaug" for causing mochitest failures @ test_pointerrawupdate_event_count.html
#   
#   This reverts commit eb27445e0352d6b4df540adb40e21f72fc052e80.
#   
#   Revert "Bug 1550462 - part 2: Make `PresShell::HandleEvent` dispatch preceding `pointerrawupdate` event r=smaug,dom-core,edgar"
# ==============================================================================

diff -r 8d25773fc82f -r f8a2a169b296 dom/events/test/pointerevents/mochitest.toml
--- a/dom/events/test/pointerevents/mochitest.toml	Fri May 16 06:20:47 2025 +0000
+++ b/dom/events/test/pointerevents/mochitest.toml	Fri May 16 09:30:09 2025 +0300
@@ -100,20 +100,6 @@
 
 ["test_pointermove_isPrimary_subsequent_pens.html"]
 
-["test_pointerrawupdate_event_count.html"]
-scheme = "https"
-skip-if = ["os == 'android'"]  # Bug 1312791
-
-["test_pointerrawupdate_event_count_touch.html"]
-scheme = "https"
-skip-if = [
-  "os == 'android'",  # Bug 1312791
-  "verify && os == 'win'",  # Bug 1659744
-  "win11_2009", # Bug 1781388
-  "os == 'win' && os_version == '11.26100' && processor == 'x86_64' && opt", # Bug 1781388
-]
-support-files = ["!/gfx/layers/apz/test/mochitest/apz_test_native_event_utils.js"]
-
 ["test_remove_frame_when_got_pointer_capture.html"]
 
 ["test_synthesized_touch.html"]